api: endpoint /accounts/{accountId} now includes transfersCount and feesCount

altergui opened the pull request:
fixes #1357

altergui committed:
api: endpoint /accounts/{accountId} now includes transfersCount and feesCount

selankon commented on api/accounts.go:
```diff
@@ -193,15 +193,27 @@ func (a *API) accountHandler(_ *apirest.APIdata, ctx *httprouter.HTTPContext) er
+
+	_, feesCount, err := a.indexer.TokenFeesList(1, 0, "", "", hex.EncodeToString(addr.Bytes()))
+	if err != nil {
+		return ErrCantFetchTokenTransfers.WithErr(err)
```
This have to be the same error type?

altergui committed:
api: hotfix 'api: paginated endpoints now return ErrAccountNotFound...'

these filters now return an empty list (not an error)
* /accounts?accountId=foobar
* /elections?electionId=foobar
* /chain/organizations?organizationId=foobar

so the errors ErrAccountNotFound, ErrElectionNotFound, ErrOrgNotFound happen only on:
* /chain/transfers?accountId=foobar
* /chain/fees?accountId=foobar
* /votes?electionId=foobar
* /elections?organizationId=foobar
